Mathlib/GroupTheory/Subgroup/Centralizer.lean

```diff
@@ -77,4 +77,22 @@ variable (H)
 theorem le_centralizer [h : H.IsCommutative] : H ≤ centralizer H :=
   le_centralizer_iff_isCommutative.mpr h
 
+/-- The homomorphism `Normalizer(H) → Aut(H)` with kernel `Centralizer(H)`. -/
```

Comment:
You're using code formatting here, but not the lean names. Probably you should do both or neither